lib/modules/platform/bitbucket-server/types.ts

```diff
@@ -70,3 +71,18 @@ export interface BitbucketErrorResponse {
 export interface BitbucketError extends HTTPError {
   readonly response: Response<BitbucketErrorResponse>;
 }
+
+export interface Comment {
+  text: string;
+  id: number;
+}
+
+export interface PullRequestCommentActivity {
+  action: 'COMMENTED';
+  commentAction: string;
+  comment: Comment;
+}
+
+export type PullRequestActivity =
+  | { action: string }
+  | PullRequestCommentActivity;
```

Comment:
Can you define them as zod schema now and only use them as type for now? so we can skip the convertion later